src/lightning/pytorch/utilities/combined_loader.py

```diff
@@ -203,6 +224,7 @@ class CombinedLoader(Iterable):
         >>> iterables = {'a': DataLoader(range(6), batch_size=4),
         ...              'b': DataLoader(range(15), batch_size=5)}
         >>> combined_loader = CombinedLoader(iterables, 'max_size_cycle')
+        >>> _ = iter(combined_loader)
```

Comment:
Instead of requiring this, should we call iter() inside `CombinedLoader.__len__` in case the iterator is still None?